dashboard/lib/pd/payment/workshop_summary.rb

```diff
@@ -46,13 +46,13 @@ def qualified?
       !payment.nil?
     end
 
-    def section_url
-      return nil unless workshop.section
-      CDO.code_org_url("/teacher-dashboard#/sections/#{workshop.section.id}", 'http:')
+    def attendance_url
+      return nil if workshop.sessions.empty?
+      "#{workshop_url}/attendance/#{workshop.sessions.first.id}"
```

Comment:
I find if to be more readable this way, since I don't expect a workshop to have zero sessions, yet I'd like to handle that potential case instead of blowing up. Unless you strongly prefer the ternary I'd prefer to leave it.